engine/api/worker/model.go

```diff
@@ -24,7 +24,7 @@ func InsertWorkerModel(db *sql.DB, model *sdk.Model) error {
 func LoadWorkerModels(db database.Querier) ([]sdk.Model, error) {
 	query := `SELECT worker_model.id, worker_model.type, worker_model.name, worker_model.image, worker_model.owner_id , "user".username
 	          FROM worker_model
-	          JOIN "user" ON "user".id =  worker_model.owner_id
+	          LEFT JOIN "user" ON "user".id =  worker_model.owner_id
```

Comment:
It's a fix, waiting for worker model right refactor. For this case, we have no user, but a group owner.